examples/stack/bqc/example_bqc_nv.py

```diff
@@ -0,0 +1,253 @@
+from __future__ import annotations
```

Comment:
Maybe it would be good to add comments to the functions in these examples what they do?

Reply:
Added some comments and a README. This also addresses your other comments below.